docs/SYNC_MODE.md

```diff
@@ -50,30 +50,22 @@ You can get all the proper inclusion metadata during discovery by calling `get_s
             replication_method=<replication_method>,
         )
 ```
-The `mdata` object can then be passed to the catalog object.
+The returned `mdata` object is in list format and can be passed directly to the Catalog object constructor.
 
-And then in the `sync` function, we'll want to iterate over only selected streams.
-We can get a list of the selected stream objects by calling `get_selected_streams()` on a singer-python `Catalog` object, as is done here:
+#### Sync
+##### Stream Selection
+Iterate over only selected streams.
+The tap can get a list of the selected stream objects by calling `get_selected_streams()` on a singer-python `Catalog` object, as is done here:
  ```
 selected_streams = catalog.get_selected_streams(state)
 
 for stream in selected_streams:
     <sync stream>
  ```
+(see [tap-trello discover.py](https://github.com/singer-io/tap-trello/blob/master/tap_trello/discover.py) for an example)
 
-## Suggested Pattern for implementing field selection
-You can get all the proper inclusion metadata during discovery by calling `get_standard_metadata()` as is done here:
-```
- mdata = metadata.get_standard_metadata(
-            schema=<schema>,
-            key_properties=<key_properties>,
-            valid_replication_keys=<replication_keys>,
-            replication_method=<replication_method>,
-        )
-```
-The `mdata` object can then be passed to the catalog object.
-
-And then in the `sync` function, we'lll want to pass every record through the transformer, as is done here:
+##### Field Selection
+The tap must pass every record through the transformer, as is done here:
```

Comment:
Maybe this could elaborate a bit like

> To apply `selected` metadata from the catalog to filter a record's fields, the supported approach is to pass every record through the transformer with a metadata dictionary, as is done here: